src/backend/controllers/tankController.js

```diff
@@ -7,82 +7,105 @@ const { validationResult } = require('express-validator');
 
 // Flow type import
 import type {
-    $Request,
-    $Response,
-    NextFunction,
-    Middleware,
-  } from 'express';
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
 
 exports.getFavorite = async (req: $Request, res: $Response) => {
-    await User.findById(req.user.id, 'favoriteTankId', function(err: Error, myUser: User){
-        if(err){
-            res.send(err);
-            console.log('could not find user');
-        } 
-        else
-            res.send(myUser.favoriteTankId);
-    });
+	await User.findById(req.user.id, 'favoriteTankId', (err: Error, myUser: User) => {
+		if(err) {
+			res.send(err);
+			console.log('could not find user');
+		} 
+		else
+			res.send(myUser.favoriteTankId);
+	});
```

Comment:
I should honestly do callbacks like these for my past API implementations. Probably will do that when I do my stuff in the morning tomorrow.